data_specs/Fractional_Cover_specs.ipynb

```diff
@@ -70,7 +77,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Coverage and metadata of Fractional Cover can be viewed on DE Africa [Metadata Explorer](https://explorer.digitalearth.africa/products/fc_ls)."
+    "Coverage and metadata can be viewed on DE Africa [Metadata Explorer](https://explorer.digitalearth.africa/products) for:\n",
```

Comment:
Do we call it "Metadata Explorer" elsewhere? I've always just called it "Explorer"...

Reply:
I think we used this term before you can download directly from Explorer... we should probably update....